world.js

```diff
@@ -18,6 +18,20 @@ import {worldMapName, appsMapName, playersMapName} from './constants.js';
 import {playersManager} from './players-manager.js';
 import * as metaverseModules from './metaverse-modules.js';
 
+setTimeout(()=>{
```

Comment:
We shouldn't run this app by default right? We can preload the module but then not instantiate it. I imagine the effects system (accessed via totum APIs) would intantiate instances of the effects app as needed and coordinate.

Reply:
Correct, its just bootstrap.